### ISSUE TITLE
Suggestion: split individual shows and squad mode wins

### ISSUE THREAD

**Issue by ksb65** [ShootMe/FallGuysStats#88]
now with the squad mode i want to keep my individual wins in the principal counter but keep the squad wins in a different one.

**Comment by mrsnarkz** [comment]
I agree. These are not true wins. I suggest filter them out. or count them only when a crown is won.

**Comment by rainyillusionist** [comment]
As an end-user I see that there is currently 2 profiles - Main and Practice. When I play Squads Show, I will switch my profile to Practice so it doesn't count into the Main wins. 

I am not entirely sure if the player.log file will contain key variables to differentiate between squad show or main show games.

**Comment by ExodiaRM** [comment]
> 
> 
> As an end-user I see that there is currently 2 profiles - Main and Practice. When I play Squads Show, I will switch my profile to Practice so it doesn't count into the Main wins.
> 
> I am not entirely sure if the player.log file will contain key variables to differentiate between squad show or main show games.

There is a line in the log file about shards, maybe something about that line can be used to detected at least the earned squad shows.

**Comment by VoidDaemon** [comment]
After a quick look at the log file the names of the squad mode games are different - they all have _squads tacked on to the end of the normal names, so it is definitely possible to detect when you are in squad mode.
